README.md

```diff
@@ -1,8 +1,39 @@
+![Contributions welcome](https://img.shields.io/badge/contributions-welcome-brightgreen.svg)
+[![License](https://img.shields.io/badge/license-MIT%20License-brightgreen.svg)](https://opensource.org/licenses/MIT)
+
+
 # GerryPy
-Geospatial Algorithm for Building Congressional Distrcits
+GerryPy is a geospatial algorithm for building congressional districts.
+
+GerryPy takes census tracts for the state of Colorado and builds the required number of congressional districts.  The algorithm attempts to make districts compact and close to the required population of 711,000.  Each algorithm attempt produces a different result.
+
+# Website
+gerrypy.herokuapp.com
+
+# Major Components
+GerryPy is built in Python and uses a PostGRES/PostGIS database.  
+
+PostGIS functions and GoogleMaps support and display the spatial data.
+
+Pyramid ORM and Bootstrap for the website.
```

Comment:
Not a ding, just a note. Pyramid is not the ORM, SQLAlchemy is. Pyramid is the Python web framework; the system serving your site to heroku. Bootstrap is the CSS framework that lets your site look pretty.